app/controllers/users_controller.rb

```diff
@@ -1,12 +1,18 @@
 class UsersController < ApplicationController
   before_action :find_user, only: [:edit, :show, :update_user_status]
-  
+
   def new
     @user = User.new
   end
 
   def create
-    @user = User.new user_params.merge(admin_status: 'applied')
+    @user = User.find_by_email(user_params[:email])
+    if @user
+      @user.applications.new(user_params[:application])
+    else
+      @user = User.new user_params
```

Comment:
The `user_params` include an `:application` key which has all the relevant information for a new application. I'm not sure about the best way to do this / name it.

What we get from simple forms on the front end is a nested hash of attributes, something like 
```
{ user: { name: 'Joe'
             application {
                  over_18: true }
             }
}
```

So the user params include the application params. I can separate them out (add a new method `application_params` that returns (basically) `params[:user][:application]` and update `user_params` to only return the user specific params then explicitly create a new user and a new application and associate them. That adds more code (and doesn't take advantage of rails's and simple_form's nesting capabilities) but if it has better clarity then I'll use it.

Thoughts?